client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -183,4 +191,113 @@ const ProfessionalEmailCard = ( {
 	return <EmailProvidersStackedCard { ...professionalEmail } />;
 };
 
-export default ProfessionalEmailCard;
+const ProfessionalEmailCardNew = ( props: EmailProvidersStackedCardProps ): ReactElement => {
+	const {
+		detailsExpanded,
+		intervalLength,
+		isDomainInCart = false,
+		onExpandedChange,
+		selectedDomainName,
+	} = props;
+	const translate = useTranslate();
+	const selectedSite = useSelector( getSelectedSite );
+	const siteSlug = selectedSite?.slug ?? '';
+	const domains = useSelector( ( state ) => getDomainsBySiteId( state, selectedSite?.ID ) );
+	const domain = getSelectedDomain( {
+		domains,
+		selectedDomainName: selectedDomainName,
+	} );
+	const emailProduct = useSelector( ( state ) =>
+		getProductBySlug( state, getTitanProductSlug( domain ) as string )
+	);
+	const provider = EmailProvider.Titan;
+
+	const cartKey = useCartKey();
+	const dispatch = useDispatch();
+	const shoppingCartManager = useShoppingCart( cartKey );
+	const [ addingToCart, setAddingToCart ] = useState( false );
+
+	const [ hiddenFieldNames, setHiddenFieldNames ] = useState< HiddenFieldNames[] >( [
+		FIELD_NAME,
+		FIELD_ALTERNATIVE_EMAIL,
+	] );
+
+	const userEmail = useSelector( getCurrentUserEmail );
+
+	const showAlternateEmailField = ( event: MouseEvent< HTMLElement > ) => {
+		event.preventDefault();
+		setHiddenFieldNames( [ FIELD_NAME ] );
+	};
+
+	const professionalEmail: ProviderCardProps = { ...professionalEmailCardInformation };
+	professionalEmail.detailsExpanded = detailsExpanded;
+
+	professionalEmail.onExpandedChange = onExpandedChange;
+	professionalEmail.priceBadge = (
+		<ProfessionalEmailPrice { ...{ domain, intervalLength, isDomainInCart } } />
+	);
+
+	const handleSubmit = getOnSubmitNewMailboxesHandler( {
+		...props,
+		dispatch,
+		domain,
+		emailProduct,
+		provider,
+		setAddingToCart,
+		shoppingCartManager,
+		siteSlug,
+	} );
+
+	const PasswordResetFieldTip = () => {
+		const translate = useTranslate();
+
+		if ( ! hiddenFieldNames.includes( FIELD_ALTERNATIVE_EMAIL ) ) {
+			return null;
+		}
+
+		return (
+			<FormSettingExplanation>
+				{ translate(
+					'Your password reset email is {{strong}}%(userEmail)s{{/strong}}. {{a}}Change it{{/a}}.',
+					{
+						args: {
+							userEmail,
+						},
+						components: {
+							strong: <strong />,
+							a: (
+								<Button
+									href="#"
```

Comment:
Why do we need to define an `href` since that's a `button` component?

Reply:
I was going for consistency with the existing view, which uses a link to render that CTA